rosdep/python.yaml

```diff
@@ -2098,7 +2114,7 @@ python-imaging:
   arch: [python2-pillow]
   debian:
     '*': [python-pil]
-    jessie:  [python-imaging]
+    jessie: [python-imaging]
```

Comment:
This change (and the ones below) still don't belong in this PR.  I actually think they are valid fixes, but please submit them separately.  This PR should just be focused on the `fastnumbers` package.